app/lib/contexts.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The model for a `context` isn't great but as it is brand new and is going to change in the future I've left it as is until such a time it needs to change. Unless there is a pressing desire for those changes to be made sooner...